unstructured/ingest/connector/vectara.py

```diff
@@ -0,0 +1,225 @@
+import json
+import typing as t
+import uuid
+from dataclasses import dataclass
+
+from unstructured.ingest.error import DestinationConnectionError
+from unstructured.ingest.interfaces import (
+    AccessConfig,
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.staging.base import flatten_dict
+
+import requests
+import datetime
+import traceback
+
+@dataclass
+class VectaraAccessConfig(AccessConfig):
+    oauth_client_id: t.AnyStr
```

Comment:
lets switch these two from t.AnyStr to:
: str = enhanced_field(sensitive=True)
And import 
`from unstructured.ingest.enhanced_dataclass import enhanced_field`
This will hide them in the logger.